sdl/main.cpp

```diff
@@ -1,10 +1,7 @@
-#include <SDL.h>
+#include "sdl_wrapper.hpp"
```

Comment:
I mildly prefer this:

```suggestion
#include "sdl/sdl_wrapper.hpp"
```

(it's a hang-over from google coding conventions)